test/old/testdir/test_filetype.vim

```diff
@@ -40,6 +40,30 @@ func Test_other_type()
   filetype off
 endfunc
 
+" If $XDG_CONFIG_HOME is set return "fname" expanded in a list.
+" Otherwise return an empty list.
+func WhenConfigHome(fname)
```

Comment:
These new functions are script-local in the patch (I guess if we wanted to be super pedantic, they should also be `abort`, as that's what `:def` functions do, but it probably doesn't matter)